61_network_comparison.Rmd

```diff
@@ -0,0 +1,323 @@
+# Network comparison {#network-comparison}
```

Comment:
same comment than in the first chapter; consider if some educational background could go to the end of the chapter as extra info; or into a separate supplementary doc that is not in table of contents but can be in OMA files and linked from here.

Reply:
I moved parts of the introduction to the end of the chapter.